src/main/java/hudson/plugins/extendedread/SystemReadPermissionEnabler.java

```diff
@@ -0,0 +1,19 @@
+package hudson.plugins.extendedread;
+
+import hudson.Extension;
+import hudson.init.InitMilestone;
+import hudson.init.Initializer;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+@Extension
+@Restricted(NoExternalUse.class)
+public class SystemReadPermissionEnabler {
+
+    @Initializer(after = InitMilestone.PLUGINS_STARTED)
+    public static void enableSystemReadPermission() {
+        if (System.getProperty("jenkins.security.SystemReadPermission") == null) {
```

Comment:
🤔 Seems like the current condition enables the feature if the property is not set. I think meant to enable the feature if the property was set.

Reply:
no this is the expected behaviour, if unset then this plugin will enable it, if set then the administrator has taken control and wants to disable it even though the plugin is installed

Reply:
Great sorry for the confusion 🌮